guides/source/getting_started.md

```diff
@@ -72,92 +72,63 @@ By following along with this guide, you'll create a Rails project called
 `blog`, a (very) simple weblog. Before you can start building the application,
 you need to make sure that you have Rails itself installed.
 
+This guide makes use of Dev Containers, which allow you to work with your Rails
+application in a container, without needing to install Rails or its dependencies
+directly on your machine. This is the fastest way to get started with Rails.
+
+If you would like to install Rails directly on your machine, you can see the
+[Installing Rails guide](installing-rails.html)
+
 NOTE: The examples below use `$` to represent your terminal prompt in a UNIX-like OS,
 though it may have been customized to appear differently. If you are using Windows,
 your prompt will look something like `C:\source_code>`.
 
-### Installing Rails
-
-Before you install Rails, you should check to make sure that your system has the
-proper prerequisites installed. These include:
-
-* Ruby
-* SQLite3
-
-#### Installing Ruby
-
-Open up a command line prompt. On macOS open Terminal.app; on Windows choose
-"Run" from your Start menu and type `cmd.exe`. Any commands prefaced with a
-dollar sign `$` should be run in the command line. Verify that you have a
-current version of Ruby installed:
-
-```bash
-$ ruby --version
-ruby 3.1.0
-```
-
-Rails requires Ruby version 3.1.0 or later. It is preferred to use the latest Ruby version.
-If the version number returned is less than that number (such as 2.3.7, or 1.8.7),
-you'll need to install a fresh copy of Ruby.
+### Installing rails-new
 
-To install Rails on Windows, you'll first need to install [Ruby Installer](https://rubyinstaller.org/).
+This guide utilizes the `rails-new` tool to create a Rails app on your machine. We will
+then run our application in a Dev Container using VSCode. Both rails-new and the
+Dev Container run in a containerized environment using Docker. So we will need to install
+VSCode and Docker as well as `rails-new`.
 
-For more installation methods for most Operating Systems take a look at
-[ruby-lang.org](https://www.ruby-lang.org/en/documentation/installation/).
+#### Installing Docker
 
-#### Installing SQLite3
+Dev Containers are run using Docker. You can install Docker by following the
+installation instructions for your operating system in the [Docker docs](https://docs.docker.com/desktop/).
 
-You will also need an installation of the SQLite3 database.
-Many popular UNIX-like OSes ship with an acceptable version of SQLite3.
-Others can find installation instructions at the [SQLite3 website](https://www.sqlite.org).
+Once Docker has been installed, open it from your application folder to begin running
+the Docker engine on your machine.
```

Comment:
Maybe we can also add something like this to connect Docker and DevContainer: 

Dev Containers will leverage Docker's containers to allow you to encapsulate your development environment and dependencies within isolated containers.